missing_element.py

```diff
@@ -0,0 +1,22 @@
+
+# Time Complexity : O(n)
+# Space Complexity : O(1)
+# Did this code successfully run on Leetcode : Yes
+# Any problem you faced while coding this : Yes
+'''
+for given numbers in the range 0 to n, we know in advance the summation of the n natural numbers is n(n+1)/2,
+so here, subtarct the sum of given elements and and subtract it from the actual sum, and that is the missing value.
+'''
+
+class Solution:
```

Comment:
Try the binary search for this problem

Reply:
But it was not given in the question that the array was sorted. Otherwise, I would have definitely used BS first. Anyway, I will update the code.

Thanks